src/ad/administration/adStatus/AdPublishedModal.js

```diff
@@ -30,7 +30,6 @@ class AdPublishedModal extends React.Component {
                 contentLabel="Fortsett"
                 onRequestClose={this.onClose}
                 closeButton
-                appElement={document.getElementById('app')}
```

Comment:
Dette er old-school. Sjekk endringen i app.js for new school – nå setter vi bare appElement én gang når appen starter